src/TicketServiceProvider.php

```diff
@@ -3,22 +3,28 @@
 namespace dnj\Ticket;
 
 use Illuminate\Support\ServiceProvider;
+use dnj\Filesystem\Contracts\IFile;
+use dnj\Filesystem\Local\File;
 
 class TicketServiceProvider extends ServiceProvider
 {
     public function register()
     {
-        $this->mergeConfigFrom(__DIR__.'/../config/ticket.php', 'ticket');
```

Comment:
Please run php-cs-fixer before you commit to keep our codes across all repositories and colleagues in a standard style.
We use `@Symfony` ruleset to keep our code global-friendly.